src/components/embed.tsx

```diff
@@ -13,22 +13,23 @@ import type { FC } from 'react';
 
 interface Props {
 	embed: Embed;
+	editions?: boolean;
```

Comment:
Could we make this explicit?

```suggestion
	editions: boolean;
```

Reply:
Good call, done.